composer.json

```diff
@@ -38,8 +39,16 @@
     "require-dev": {
         "ext-mcrypt": "*",
         "mikey179/vfsStream": "~1.0",
-        "symfony/filesystem": "~2.3",
-        "symfony/process": "~2.3"
+        "symfony/filesystem": "^4.3",
+        "symfony/form": "^4.3",
+        "symfony/process": "^4.3",
```

Comment:
Why do we need to include all these dependencies suddenly? Seems a bit strange also to have something like messenger or forms as dev requirements? Or is it just me missing something?

Reply:
I included this updated branch in our update PR for MIA and it complained that this component didn't have these dependencies.

Reply:
Should I remove them?

Reply:
> Should I remove them?

I am not sure.